releasenotes/notes/iast-fix-cookie-report-573f444827a1f3f5.yaml

```diff
@@ -0,0 +1,6 @@
+---
+fixes:
+  - |
+    Vulnerability Management for Code-level (IAST): IAST is independent of Appsec; ``DD_IAST_ENABLED`` doesn't require 
+    ``DD_APPSEC_ENABLED``. With that, this fix introduces a small refactor when both IAST and ASM check the 
+    request cookies.
```

Comment:
Can we make this release note more direct. From the perspective of ddtrace users it is not clear what changed in this PR. The note about the request cookies fix could be it's own release note.